### PR TITLE
#676: Restore directions for developing in a local environment

### PR DESCRIPTION
**Pull request by ianperera**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)
[#676](https://app.zenhub.com/workspaces/va-notify-620d21369d810a00146ed9c8/issues/department-of-veterans-affairs/notification-api/676)
## Type of change

Please delete options that are not relevant.

- [x] Documentation changes only

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,38 +96,146 @@ Running `flask db migrate` on the container ci_app_1 errors because the files in
...
+  `brew install pyenv`
+
+ 2. Install Python 3.6.10 (or whatever version is specified in .python-version)
+ Then follow from instructions for rest of pyenv setup, [see step 3 here](https://github.com/pyenv/pyenv#basic-github-checkout)
+
+ Note: For MacOS devs who are using Big Sur, Monterey, standard pyenv python installation will be failed in most case. I found [this solution](https://github.com/pyenv/pyenv/issues/2143#issuecomment-1070640288) so only 3.7.13, 3.8.13, 3.9.11 and 3.10.3 works fine.
+
+ `pyenv install 3.6.10`
+
+ 3. If you expect no conflicts, set `3.6.10` as you default
```
Comment (EvanParish) [inline README.md:153]: Should these be `3.8` instead of `3.6`?

Comment (cris-oddball) [reply README.md:153]: Yes. Or a more specific version of 3.8.  3.8.14 was just released earlier this month, and 3.8 itself was released in 2019. Have we standardized on any specific version?

Comment (cris-oddball) [reply README.md:153]: FYI on my Mac M1 (silicon), I'm using 3.8.13, which is what that note says on line 149, so I think that should be the version.

File: README.md [hunk 2]
```diff
@@ -99,38 +96,146 @@ Running `flask db migrate` on the container ci_app_1 errors because the files in
...
+  `brew install pyenv`
+
+ 2. Install Python 3.8.13 (or whatever version is specified in .python-version)
+ Then follow from instructions for rest of pyenv setup, [see step 3 here](https://github.com/pyenv/pyenv#basic-github-checkout)
+
+ Note: For MacOS devs who are using Big Sur, Monterey, standard pyenv python installation will be failed in most case. I found [this solution](https://github.com/pyenv/pyenv/issues/2143#issuecomment-1070640288) so only 3.7.13, 3.8.13, 3.9.11 and 3.10.3 works fine.
+
+ `pyenv install 3.8.13`
+
+ 3. If you expect no conflicts, set `3.8.13` as you default
```
Comment (cris-oddball) [inline README.md:153]: you > your ;)

**Review by EvanParish — APPROVED** [review]

**Review by cris-oddball — APPROVED** [review]

**Pull request merged**